samples/sampler/stories/default/Arc.js

```diff
@@ -0,0 +1,33 @@
+import {mergeComponentMetadata} from '@enact/storybook-utils';
+import {select, number} from '@enact/storybook-utils/addons/knobs';
+import React from 'react';
+import {storiesOf} from '@storybook/react';
+
+import Arc, {ArcBase} from '@enact/agate/Arc';
+
+Arc.displayName = 'Arc';
+const Config = mergeComponentMetadata('Arc', ArcBase, Arc);
+
+// Set up some defaults for info and knobs
+const prop = {
+	colors: ['', '#000000', '#FDC902', '#986AAD']
+};
+
+storiesOf('Agate', module)
+	.add(
+		'Arc',
+		() => (
+			<div style={{width:'40%', height:'40%'}}>
+				<Arc
+					color={select('color', prop.colors, Config)}
+					endAngle={number('endAngle', Config, {range: true, min: 0, max: 360})}
+					radius={number('radius', Config)}
+					startAngle={number('startAngle', Config, {range: true, min: 0, max: 360})}
```

Comment:
Maybe setting ` startAngle` and `endAngle` as simple inputs may be a better choice. It's easier for an user to just type the wanted value rather than selecting it form a range. What do you think?

Reply:
well, both of them have a lower limit and a higher limit( 0  - 360 ). That is why I chosed a range in the story

Reply:
I see. Ok, let's go with this approach.